app/views/admin/organizations/_form.html.haml

```diff
@@ -1,118 +1,88 @@
-- if @organization.errors.any?
-  #alert-block
-    .alert.alert-error
-      %h4
-        Please fix the following
-        = pluralize(@organization.errors.count, "error")
-      %ul
-        - @organization.errors.full_messages.each do |msg|
-          %li
-            = msg
+.card.bg-light
```

Comment:
Why did you decide to put this in a card?

Reply:
I've just been putting a lot of the forms in cards, but I guess in the particular case there are other fields that render. I can remove it if it makes you feel better. I just got into a habit of doing it.

Reply:
It wasn't bad before, it just didn't seem like it was helping the flow of the page.

I think I like it more like this, let me know if you disagree 🙂

Reply:
No, I agree with you. I think on the pages where the form is this only thing that renders it makes sense to do it in a card, because it looks nice, but when this form renders there's possible a lot of other stuff rendering with it and it does get a little chunky.